app/views/boxes/_barcode_card.haml

```diff
@@ -0,0 +1,19 @@
+.barcode-card.is-box
+  .barcode-content
+    .barcode(style="float:right")
+      = image_tag barcode_svg_data_url(@box.uuid)
```

Comment:
It should use the PNG generator. The SVG looks much better on screen, but the PNG has been validated on the printer!